guides/plugins/plugins/storefront/add-icons.md

```diff
@@ -0,0 +1,85 @@
+# Add custom icons
+
+## Overview
+
+In this guide you will learn how to use the icon renderer component as well as adding custom icons.
+
+## Prerequisites
+
+In order to follow this guide easily, you first need to have a functioning plugin installed. Head over to our 
+[plugin base guide](../plugin-base-guide.md) to create a plugin, if you don't know how it's done yet.
+Also knowing and understanding SCSS will be quite mandatory to fully understand what's going on here.
+Furthermore, it might be helpful to read the guide on how to [handle own assets](./add-custom-assets.md) in your 
+plugin before you start with this one.
+
+## Adding icon
+
+In order to add any icons to the Storefront, you use our `sw_icon` twig action. This way, an icon of choice is 
+displayed in the Storefront.
+
+Needless to say, the first step is saving your image somewhere in your plugin Shopware can find it. The default path 
+for icons is thw following:
+```
+<YourTheme>/src/Resources/app/storefront/dist/assets/icon/default
+````
+
+You can also provide "solid" icons or any other custom pack names which can be configured later with the `pack` 
+parameter. You can do that by creating a folder with the pack name:
+```
+<YourTheme>/src/Resources/app/storefront/dist/assets/icon/<pack-name>
+```
+
+By default, shopware looks inside the "default" 
+folder.
+
+```
+{% sw_icon 'done-outline-24px' style {
+    'namespace': 'TestTheme'
+} %}
+```
+
+You surely want to add your own custom icons. In this case, the `namespace` parameter is the most important one to configure.
+In there, you need to set the name of the theme in which the icon is searched for by its name.
```

Comment:
Does this only work for themes?
If so, ignore my other comments about changing `<YourTheme>`.

But in that case I wonder if this should be part of the theme guides?

Another idea:
Can we link the same file twice in the Summary.MD, once for plugins and once for themes? 🤔

Reply:
I think I wrote that out of habit. :) Howver, I'd check that bach with @tobiasberge . Is it the same with themes and plugins?

Reply:
Yes I guess it should also work normally for plugins as well. So it could also be inside the plugin guides, but maybe the use case is more common for themes to add custom icons.

Reply:
Ok, this way linking it twice is a good idea :)